### ISSUE TITLE
Plone 5 deprecation warning

### ISSUE THREAD

**Issue by tisto** [plone/plone.restapi#975]
Follow up on https://github.com/plone/Products.CMFPlone/issues/3130:

```
/home/vincentfretin/workspace/buildout.coredev-5.2/src/plone.restapi/src/plone/restapi/controlpanels/__init__.py:6: DeprecationWarning: ILanguageSchema is deprecated. It has been moved to plone.i18n.interfaces, import from there instead.
  from Products.CMFPlone.interfaces.controlpanel import ILanguageSchema
```

**Comment by tisto** [comment]
https://pypi.org/project/plone.i18n/4.0.4/ introduced ILanguageSchema.

**Comment by mauritsvanrees** [comment]
Partially fixed by @tisto in cec8f442d3725b4d68eff7ac3ab846c39ae9c414 and/or e4a2ad0285a85c5eed54d49c10d8390bef4ade57 (released in 6.13.6).

`ILanguageSchema` from `CMFPlone` is still there in the tests though.

And we use deprecated zope.site.hooks.

I am preparing a small PR: https://github.com/plone/plone.restapi/pull/988

**Comment by mauritsvanrees** [comment]
My PR was merged.
